src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -449,6 +490,7 @@ MulticopterPositionControl::MulticopterPositionControl() :
 	_vel_sp_prev.zero();
 	_vel_err_d.zero();
 	_curr_pos_sp.zero();
+	_stick_input_xy_prev = matrix::Vector2f(0.0f, 0.0f);
```

Comment:
use the zero initialiser: `_stick_input_xy_prev{}`. The code above constructs a zero-object matrix::Vector, then copies its contents into the zero-initialised _stick_input_xy_prev. Math example:

stick_input = 0;
zero_obj = 0;

stick_input = zero_obj;

On the other hand doing _stick_input_xy_prev does what you expect:

stick_input = 0;